src/plugin/regions.js

```diff
@@ -622,6 +628,9 @@ export default class RegionsPlugin {
         this.params = params;
         this.wavesurfer = ws;
         this.util = ws.util;
+        this.util.getRegionSnapToGridValue = value => {
```

Comment:
why put this in `util`?

Reply:
I need access to it from the Region and from the RegionsPlugin. Is there a better place?